src/client/components/EditUserForm/CollaboratorPermissions/CollaboratorPermissions.scss

```diff
@@ -0,0 +1,12 @@
+.edit-user__form-item-permissions {
+  height: auto;
+}
+
+.edit-user__form-field-premissions {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 12px;
+      font-size: 12px;
```

Comment:
isnt this auto formatted?

Reply:
i don't know what happened with mi ide. I save it again and it auroformatted-it.

Reply:
Don't you use pre commit hook ?

Reply:
yes, I do

Reply:
can u check ?